worker/backup_handler.go

```diff
@@ -62,11 +62,6 @@ const (
 )
 
 func createBackupFile(h UriHandler, uri *url.URL, req *pb.BackupRequest) (io.WriteCloser, error) {
-	if !h.DirExists("./") {
-		if err := h.CreateDir("./"); err != nil {
-			return nil, errors.Wrap(err, "while creating backup file")
-		}
-	}
```

Comment:
This piece of code was no-op because it would have failed already if the `backup` dir was not present when checking for manifest file. Is that correct?

Reply:
exactly .